install_radar_deps.py

```diff
@@ -0,0 +1,289 @@
+#!/bin/python3
+
+"""
+Copyright SuperDARN Canada 2020
```

Comment:
Not sure if you have this somewhere else, but it might be nice to have a comment at the top of this script specifying which Linux distributions this script has been tested on and can handle.